packages/annotorious-core/src/lifecycle/Lifecycle.ts

```diff
@@ -35,8 +35,8 @@ export const createLifecyleObserver = <I extends Annotation, E extends unknown>(
     const callbacks = observers.get(event);
     if (callbacks) {
       const idx = callbacks.indexOf(callback);
-      if (idx > 0)
-        callbacks.splice(callbacks.indexOf(callback), 1);
+      if (idx !== -1)
+        callbacks.splice(idx, 1);
```

Comment:
I'm really not sure what I must have smoked that day :-D

Reply:
I got shot myself with the same exact comparison more times I would like to admit 😆